test/metabase/query_processor/card_test.clj

```diff
@@ -41,12 +41,13 @@
                       Dashboard [dash {:cache_ttl 1338}]
                       Card [card {:database_id (u/the-id db)}]]
         (is (= (* 3600 1338) (:cache-ttl (#'qp.card/query-for-card card {} {} {} {:dashboard-id (u/the-id dash)}))))))
-    (testing "multiple ttl, db wins"
+    (testing "multiple ttl, db ttl does not take effect on OSS so nil result"
+      ;; corresponding EE test in metabase-enterprise.advanced-config.caching-test
       (mt/with-temp* [Database [db {:cache_ttl 1337}]
                       Dashboard [dash]
                       Card [card {:database_id (u/the-id db)}]]
-        (is (= (* 3600 1337) (:cache-ttl (#'qp.card/query-for-card card {} {} {} {:dashboard-id (u/the-id dash)}))))))
-    (testing "no ttl, nil res"
+        (is (= nil (:cache-ttl (#'qp.card/query-for-card card {} {} {} {:dashboard-id (u/the-id dash)}))))))
+    (testing "no ttl, nil result"
       (mt/with-temp* [Database [db]
                       Dashboard [dash]
                       Card [card {:database_id (u/the-id db)}]]
```

Comment:
I think this is missing an assertion. probably just need to check for nil again

Reply:
it's the line below, just cut off by the diff

Reply:
🤦‍♂️